cmd/sigclient.go

```diff
@@ -110,6 +111,47 @@ var metricsQueryCmd = &cobra.Command{
 	},
 }
 
+var cmdWrap wrapper
+
+type wrapper struct {
+	err error
+}
+
+// RunE fails to proceed further in case of error resulting in not executing PostRun actions
+func (w *wrapper) Run(f func(cmd *cobra.Command, args []string) error) func(cmd *cobra.Command, args []string) {
+	return func(cmd *cobra.Command, args []string) {
+		err := f(cmd, args)
+		w.err = err
+	}
+}
+
+var metricsQueryValidateOutputCmd = &cobra.Command{
```

Comment:
can this be merged with the existing `metricsQueryCmd`? can `StartMetricsQuery` add an argument for validation?